pkg/controller/vault.go

```diff
@@ -460,9 +460,8 @@ func (v *vaultSrv) GetRBACRolesAndRoleBindings() ([]rbac.Role, []rbac.RoleBindin
 func (v *vaultSrv) GetRBACClusterRoleBinding() rbac.ClusterRoleBinding {
 	return rbac.ClusterRoleBinding{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      v.vs.Name + "-k8s-token-reviewer",
-			Namespace: v.vs.Namespace,
-			Labels:    v.vs.OffshootLabels(),
+			Name:   v.vs.Namespace + "-" + v.vs.Name + "-k8s-token-reviewer",
```

Comment:
Can you please use 
`meta_util.NameWithPrefix(v.vs.Namespace + "-" + v.vs.Name, "k8s-token-reviewer")`